Docker-Swarm-deployment/single-node/lip/tests/example-config.json

```diff
@@ -1,9 +1,10 @@
 {
+    "createVhost" : "True or False, if vhost is required to create",
     "username": "<rmq user>", 
     "password": "<rmq password>", 
     "host": "<rmq host>", 
     "port": "<rmq amq port>", 
-    "hport": "<rmq https port>",
+    "hport": "<rmq https port, required only if createVhost is true >",
```

Comment:
use proper config fields, should be clear by just reading it

- use ``httpsPort`` instead of ``hport``
- use ``amqpsPort`` instead of ``port`` --> also its better to connect and test over amqps .